string-manipulation-1/src/reverse-word.js

```diff
@@ -0,0 +1,9 @@
+/* exported reverseWord */
+function reverseWord(word) {
+  let str = '';
+  for (let i = word.length - 1; i >= 0; i--) {
+    str += word[i];
+  }
+  return str;
+}
+console.log(reverseWord('LearningFuze'));
```

Comment:
Delete any console logs. These are only used for testing / debugging purposes

Reply:
Oh ok I won't , but on exercises which already pushed I think you will see some console.los as well. From now on I will keep in mind and try not to log in the file . Thank you for your very detailed oriented feedbacks , I really appreciate  it!